Gems/EditorPythonBindings/Code/Tests/PythonTupleTests.cpp

```diff
@@ -238,12 +238,12 @@ namespace UnitTest
                 test = azlmbr.object.create('PythonReflectionTupleTypes')
 
                 # Create a tuple with default values - note the odd naming syntax :(
-                test_tuple = azlmbr.object.create('tuple<bool int float >')
+                test_tuple = azlmbr.object.create('AZStd::tuple<bool, int, float>')
```

Comment:
The " - note the odd naming syntax :(" can be removed now, since it's now a normal and good naming syntax. :)

Reply:
Will remove the blurb